assets/js/components/SaptuneDetails/SaptuneDetails.jsx

```diff
@@ -0,0 +1,193 @@
+import React from 'react';
+
+import { find, map } from 'lodash';
+
+import { Link } from 'react-router-dom';
+
+import BackButton from '@components/BackButton';
+import ListView from '@components/ListView';
+import PageHeader from '@components/PageHeader';
+
+import SaptuneTuningState from './SaptuneTuningState';
+import SaptuneVersion from './SaptuneVersion';
+
+const renderService = (serviceName, services) => {
+  const currentService = find(services, { name: serviceName });
+
+  if (!currentService.enabled) {
+    return <span>-</span>;
+  }
+
+  return (
+    <span>
+      {currentService.enabled}/{currentService.active}
+    </span>
+  );
+};
+
+const renderNote = (noteID) => (
+  <Link
+    key={noteID}
+    className="text-jungle-green-500 hover:opacity-75"
+    to={`https://launchpad.support.sap.com/#/notes/${noteID}`}
+  >
+    {noteID}
+  </Link>
+);
+
+const renderNotes = (notes) => {
+  if (notes.length === 0) {
+    return <span>-</span>;
+  }
+  return notes.map((noteID, index) => [index > 0 && ', ', renderNote(noteID)]);
+};
+
+const renderSolution = ({ id, notes, partial }) => (
+  <span>
+    {id} ({renderNotes(notes)}
+    {partial ? ' -> Partial' : ''})
```

Comment:
Maybe we need to remove the space in ` -> Partial`. Just be sure to only have 1 space there